src/reynir_correct/main.py

```diff
@@ -335,14 +337,41 @@ def sentence_stream() -> Iterator[List[CorrectToken]]:
             )
             for ann in a
         ]
+        if args.text:
+            if sent.tree is None:
+                # No need to do more, no grammar errors have been checked
+                print(cleaned, file=args.outfile)
+            else:
+                # We know we have a sent tree, can use that
+                # Use toklist and a
```

Comment:
Something missing from the end of the comment

Reply:
Done